pkg/OutputColorizer.nuspec

```diff
@@ -20,6 +20,6 @@
     </metadata>
     <files>
         <file src="net45\OutputColorizer.dll" target="lib\net45" />
-        <file src="Core\OutputColorizer.dll" target="lib\netstandard1.5" />
+        <file src="..\OutputColorizer.Core\bin\Debug\netstandard1.5\OutputColorizer.dll" target="lib\netstandard1.5" />
```

Comment:
Doesn't this mean that the output for the core binaries is going to be under the project?